### DISCUSSION TITLE
RTL support

### DISCUSSION THREAD

**Discussion by puraminy** [MrGreensWorkshop/MrGreen-JekyllTheme#103] [Feature requests]
Thank you for your great work!

I see your theme supports multilingual langauges, and some language are from right to left. Sometimes chaining the direction of page can works or it may need more settings like floats. Anyways, I would like to know how can I make it right to left and if you have or will add a setting for that? 

Regards,

**Comment by MrGreensWorkshop** [comment]
Hi, Since I don't have any experience about right to left languages, I don't think I can add it myself. If any one wants to add, PRs are welcome.

**Comment by puraminy** [comment]
Thanks, I am also new to Jekyll and how your settings works, but I'll give it a try.

Initially I want a way to add one atrribute to the header html tag (that applies to all pages) 

```
<html dir="rtl" >
```
If you know how and where I can do it please tell me

**Comment by MrGreensWorkshop** [comment]
If you know where to add then you should search in the source code. My preferred tool is VS Code.

Why don't you try to search html tag? "<html" 

to add options check how other settings added

ex, search for color_scheme_default_dark

{{ site.data.conf.main.color_scheme_default_dark }}

**Comment by puraminy** [comment]
Thanks, yes it is in _layout/default.md
I declared a variable like `lng_code` namded `lng_dir` and  put it in the tag. I am working to put some conditions for some layouts like the main-wrapper or side nave that must be go to the right, etc.

I have other works too, but when I completed this I offer my modifications...
